int32 id
---
string room
float32 x
float32 y
RoomConnection[] connections
